src/fields/fieldPassword.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="jade">
-	input.form-control(type="password", v-model="value", :readonly="schema.readonly", :disabled="disabled", :placeholder="schema.placeholder")
+	input.form-control(type="password", v-model="value", :autocomplete="schema.autocomplete", :disabled="disabled", :readonly="schema.readonly")
 </template>
```

Comment:
Sorry the [french documentation](https://developer.mozilla.org/fr/docs/Web/HTML/Element/Input) was obsolete and wasn't listing `password` and `number` as compatible with `placeholder`. I will fix that.